src/main/java/edu/greenblitz/tobyDetermined/commands/MultiSystem/FullGrip.java

```diff
@@ -0,0 +1,33 @@
+package edu.greenblitz.tobyDetermined.commands.MultiSystem;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.tobyDetermined.commands.intake.extender.ExtendRoller;
+import edu.greenblitz.tobyDetermined.commands.intake.extender.RetractRoller;
+import edu.greenblitz.tobyDetermined.commands.intake.roller.RunRoller;
+import edu.greenblitz.tobyDetermined.commands.rotatingBelly.RotateOutDoorDirection;
+import edu.greenblitz.tobyDetermined.commands.telescopicArm.GoToPosition;
+import edu.greenblitz.tobyDetermined.subsystems.RotatingBelly.RotatingBelly;
+import edu.greenblitz.tobyDetermined.subsystems.intake.IntakeRoller;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class FullGrip extends SequentialCommandGroup {
+
+    public FullGrip() {
+        //FIRST -> open the gripper and run it until an object is inside, along with to change the arm position
+        addCommands(
+                new ParallelCommandGroup(
+                        new ExtendRoller().alongWith(new RunRoller())
+                                .until(() -> RotatingBelly.getInstance().isObjectIn() || IntakeRoller.getInstance().isObjectIn()),
```

Comment:
the rotating belly interupt should be handled in FullGripAndPutInClaw

Reply:
tal - this makes sure you dont accidentally take 2 items